source/databricks/calculation_engine/tests/calculation_input/test_table_reader.py

```diff
@@ -105,6 +103,31 @@ def _create_charge_master_period_row() -> dict:
     }
 
 
+def _map_metering_point_type_and_settlement_method(df: DataFrame) -> DataFrame:
+    """
+    Maps metering point type and settlement method to the correct values
+    Only supports consumption and flex
```

Comment:
Is it just because the others aren't currently required?
```suggestion
    Currently only supports consumption and flex
```

Reply:
yes